html-css/week2/Neepun_orderPage/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/styles.css" />
+  <title>Document</title>
+</head>
+
+<body>
+  <header class="header">
+    <h1>Payment Header</h1>
+    <p>Plesae Enter Payment Details</p>
+
+  </header>
+  <main class="main">
+    <section class="paymentSection">
+      <!-- Order Summary Container -->
+      <div class="summaryContainer">
+        <div class="orderSummary">
+          <h2>Order Summary</h>
+            <div class="amountContainer">
+              <div class="totalContainer">
+                <p>Total</p>
+                <p>$333 / month</p>
+              </div>
```

Comment:
Those are definitely not separate paragraphs, considering how short the copy is. I think it could be better marked like this:
```suggestion
              <p class="totalContainer">
                <span>Total</span>
                <span>$333 / month</span>
              </p>
```